ts/Extensions/ParallelCoordinates.ts

```diff
@@ -678,19 +678,24 @@ namespace ParallelAxis {
         }
 
         if (chart && chart.hasParallelCoordinates && !axis.isXAxis) {
-            const index = parallelCoordinates.position,
-                currentPoints: Array<Point> = [];
+            const index = parallelCoordinates.position;
+            let currentPoints: Array<number|null> = [];
 
             axis.series.forEach(function (series): void {
-                if (
-                    series.visible &&
-                    defined((series.yData as any)[index as any])
-                ) {
-                    // We need to use push() beacause of null points
-                    currentPoints.push((series.yData as any)[index as any]);
+                if (series.yData && series.visible) {
+                    const y = (series.yData)[index as any];
```

Comment:
I believe this `as any` can be removed here by checking if `index` is a number type.